opentreemap/treemap/models.py

```diff
@@ -507,6 +507,7 @@ class InstanceUser(Auditable, models.Model):
     role = models.ForeignKey(Role)
     reputation = models.IntegerField(default=0)
     admin = models.BooleanField(default=False)
+    last_seen = models.DateField(null=True, blank=True)
```

Comment:
`InstanceUser` is `Auditable` so I think we need to explicitly exclude this field from audit tracking to avoid creating an audit row on every authenticated page request.